tests/test_diagnosis_routes.py

```
@@ -0,0 +1,142 @@
+    transport = ASGITransport(app=app)
+    async with AsyncClient(transport=transport, base_url="http://test") as ac:
+        resp = await ac.get("/ai/diagnosis/drowsiness", params={"deviceUid": device_uid})
+
+    assert resp.status_code == 200
+    data = resp.json()
+    assert data["success"] is True
+    assert data["isDrowsinessDrive"] is True # 0.5 이하는 True 반환
+    assert "detectionTime" in data
+
+@pytest.mark.asyncio
+async def test_model_not_loaded():
```

Ruff (0.11.9)

Missing return type annotation for public function `test_model_not_loaded`

Add return type annotation: `None`

(ANN201)